lib/client/client.go

```diff
@@ -548,58 +550,221 @@ func (client *NodeClient) scp(scpCommand scp.Command, shellCmd string, errWriter
 	return trace.Wrap(err)
 }
 
+func proxyConnection(client *NodeClient, incoming net.Conn, remoteAddr string) {
+	defer incoming.Close()
+	var (
+		conn net.Conn
+		err  error
+	)
+	log.Debugf("nodeClient.proxyConnection(%v -> %v) started", incoming.RemoteAddr(), remoteAddr)
+	for attempt := 1; attempt <= 5; attempt++ {
+		conn, err = client.Client.Dial("tcp", remoteAddr)
```

Comment:
I believe this needs to be a `direct-tcpip` request on the channel, so it should be `DialTCP`.